group/edwards25519/const.go

```diff
@@ -6,21 +6,31 @@ package edwards25519
 
 import (
 	"math/big"
-
-	"gopkg.in/dedis/kyber.v1/group/mod"
 )
 
 // prime modulus of underlying field = 2^255 - 19
 var prime, _ = new(big.Int).SetString("57896044618658097711785492504343953926634992332820282019728792003956564819949", 10)
 
 // prime order of base point = 2^252 + 27742317777372353535851937790883648493
-var primeOrder, _ = new(mod.Int).SetString("7237005577332262213973186563042994240857116359379907606001950938285454250989", "", 10)
+// XXX this should probably just be a big.Int
+var primeOrder, _ = new(big.Int).SetString("7237005577332262213973186563042994240857116359379907606001950938285454250989", 10)
```

Comment:
With reference to the comment `this should probably just be a big.Int` I thought that there is another way to instantiate it. So I didn't understand the comment. Perhaps just take the old comment, which looks clearer to me?